schema.prisma

```diff
@@ -265,6 +265,15 @@ model Country {
   @@map("countries")
 }
 
+model CampaignDocumentRole {
+  id     String @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  createdAt          DateTime            @default(now()) @map("created_at") @db.Timestamptz(6)
+  updatedAt          DateTime?           @updatedAt @map("updated_at") @db.Timestamptz(6)
+  name  String
+  description  String
+  @@map("bootcamp")
```

Comment:
This model should not have name `bootcamp`

Reply:
:ok_hand: